vita3k/renderer/src/vulkan/renderer.cpp

```diff
@@ -32,7 +32,9 @@
 #include <SDL_vulkan.h>
 
 #ifdef __APPLE__
-#include <mvk_config.h>
+#include <MoltenVK/mvk_config.h>
+typedef VkResult (VKAPI_PTR *PFN_vkSetMoltenVKConfigurationMVK)(VkInstance ignored, const MVKConfiguration* pConfiguration, size_t* pConfigurationSize); // #include <MoltenVK/mvk_deprecated_api.h>
+
```

Comment:
KhronosGroup deprecated SetMoltenVKConfigurationMVK and changed it to use the standard Vulkan `VK_EXT_layer_settings` extension. But `VK_EXT_layer_settings` is introduced by Vulkan API revision 1.3.275. So to adapt this, we need to update Vulkan-Header and related things.

I couldn't use `mvk_deprecated_api.h` header because of conflict with our PTR class implementation and macOS's internal PTR typedef.

Reply:
I can update the vulkan headers, I'll do it

Reply:
Changed to use `VK_EXT_layer_settings` when setting full image view swizzle.
Ref. https://github.com/KhronosGroup/MoltenVK/blob/main/Docs/MoltenVK_Configuration_Parameters.md / https://vulkan.lunarg.com/doc/sdk/1.3.290.0/windows/layer_configuration.html